### PR TITLE
Feature 156874214 Email Confirmation Status

### PR DESCRIPTION
**Pull request by roedesh**
Here is the PR for the changes made to the email confirmation status feature, shown yesterday during the demo.

Changes:
- Added a link to the warning message that will instantly resend the email, instead of redirecting you to the Devise form
- Made some changes to the _message.html partial and the close-message javascript function. Now flash messages can be closed individually
- Made soms changes to add_transaction spec to work with the new _message.html partial

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -14,6 +14,8 @@
   get :users, to: 'users#autocomplete_search', as: :users_autocomplete
 
   get :settings, to: 'users#edit', as: :user
+  get :resend_email_confirmation, to: 'users#resend_email_confirmation',
```
Comment (ariejan) [inline config/routes.rb]: This should not be a GET request IMHO.

http://thedailywtf.com/articles/WellIntentioned-Destruction

Comment (roedesh) [reply config/routes.rb]: I'll change it to a POST request

**Comment by ariejan** [comment]
Or a PUT :-)

GET: Retrieve shit
POST: Create new shit
PUT: Update shit
DELETE: Throw away shit

**Comment by roedesh** [comment]
Changed it. I chose POST since we CREATE some shit (a new confirmation token and a new email) :-)

**Pull request merged**